src/app/space/create/deployments/services/deployments.service.ts

```diff
@@ -157,25 +157,25 @@ export interface MultiTimeseriesData {
   end: number;
 }
 
-export interface TimestampedCpuStats {
+export interface TimestampedCpuStat {
   data: CpuStat;
   timestamp: number;
 }
 
-export interface TimestampedMemoryStats {
+export interface TimestampedMemoryStat {
   data: ScaledMemoryStat;
   timestamp: number;
 }
 
-export interface TimestampedNetworkStats {
+export interface TimestampedNetworkStat {
   data: { sent: ScaledNetworkStat, received: ScaledNetworkStat };
   timestamp: number;
 }
```

Comment:
TimestampedCpuStat is now redundant since CpuStat contains an optional timestamp, and same for the Memory and Network Stat subinterfaces. Please follow up with a second commit to squash these down, replacing TimestampedCpuStat with simply CpuStat and adapting any consumers.

Reply:
Sounds good, I'll have this updated shortly.